adapters/Snap/SnapAdapter/GADSAKAdViewContainer.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import <GoogleMobileAds/GoogleMobileAds.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface GADSAKAdViewContainer : NSObject <GADMediationBannerAd>
```

Comment:
To be consistent with the other adapters, the class name should start with `GAD`. Therefore we can probably rename it to something like: `GADMediationSnapBanner`.

Reply:
I don't think this naming is consistent throughout.
A similar class is named `GADFBBannerRenderer` in Facebook adapter implementation